lookout/style/typos/tests/test_datasets_preparation.py

```diff
@@ -50,5 +53,17 @@ def test_prepare_data_from_file(self):
         shutil.rmtree(temp_dir)
 
 
+class FasttextTest(unittest.TestCase):
+    def test_get_fasttext_model(self):
+        data = pandas.read_csv(str(pathlib.Path(__file__).parent / "prepared_data.csv.xz"),
```

Comment:
I see that we use `pathlib.Path(__file__).parent` here and in `DATA_DIR`, may be worth defining `BASE_DIR`